contracts/core/CoreV2.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import {Vcon} from "../vcon/Vcon.sol";
+import {IVolt, Volt, IERC20} from "../volt/Volt.sol";
+import {ICoreV2} from "./ICoreV2.sol";
+import {PermissionsV2} from "./PermissionsV2.sol";
+import {Initializable} from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+
+/// @title Source of truth for VOLT Protocol
+/// @author Volt Protocol
+/// @notice maintains roles, access control, Volt, Vcon, and the Vcon treasury
+contract CoreV2 is ICoreV2, PermissionsV2 {
+    /// @notice the address of the Volt contract
+    IVolt public immutable override volt;
```

Comment:
are we sure there won't be further VOLT migrations ?

Reply:
Good point, will make volt a mutable state variable